src/components/KMethod/KMethod.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <KBadge
+    :background-color="backgroundColor || methodBadgeColors[method].backgroundColor"
+    class="k-method"
+    :class="`method-${method}`"
+    :color="color || methodBadgeColors[method].color"
+    max-width="auto"
+    :shape="appearance === 'rectangular' ? 'rectangular' : 'rounded'"
+  >
+    {{ methodLabel }}
+  </KBadge>
+</template>
+
+<script lang="ts">
+import { PropType, computed } from 'vue'
+import type { Method, MethodAppearance } from '@/types'
+
+export const methods: Method[] = ['get', 'post', 'put', 'patch', 'delete', 'head', 'options', 'trace', 'connect', 'custom']
+
+export const shapes: MethodAppearance[] = ['rounded', 'rectangular']
```

Comment:
question: why are these defined here instead of the types file you created? We shouldn't be exporting constants from the component. 

There are existing examples of this pattern in the other components